site/content/api/Tenant-API.md

```diff
@@ -100,15 +98,21 @@ The response message's *status* property may contain the following codes:
 | Code | Description |
 | :--- | :---------- |
 | *200* | OK, the payload contains the tenant information for the requested tenant. |
-| *404* | Not Found, there is no tenant identified with the given *tenant-id*. |
+| *404* | Not Found, there is no tenant identified with the given *tenant_id*. |
 
 For status codes indicating an error (codes in the `400 - 499` range) the message body MAY contain a detailed description of the error that occurred.
 
 ## Update Tenant
 
 Clients use this command to *update* information about an already existing tenant. All of the information that has been previously provided for the tenant gets *replaced* with the information contained in the request message.
 
-This operation is *optional*, implementors of this API may provide other means for updating tenant information, e.g. a RESTful API or a configuration file.
+With the update of the tenant entity it is e.g. possible to enable new adapters for a given tenant or configure the maximum allowed message rate.
```

Comment:
+1 for adding the outline of some possible effects.

What still should be improved IMHO is a clearer distinction between:
- this document *only* defines the handling of tenants, including the format and the names of involved data structures
- the impacts on the overall Hono system, however, are outlined and there are at least these two different kinds of impacts that are described :
  - how e.g. protocol adapters need to use this API for a senseful implementation (like considering the enabled flag of the tenant)
  - and what are more complicated impacts of this API (like removing rights for a tenant are probably not considered at once), but that this API does NOT solve these problems, only pointing them out

That would make a more straightforward read for my taste.

So, I propose you structure these points more according to these points. WDYT?

Reply:
That's a good point :-)
What if I add another paragraph in the introduction section that this document just describes the API (your first bullet point) and that in here we're just outlining *possible consequences* to the whole system. 

Effects to the whole system and how / if protocol adapters are required to use the tenant api I would put on a separate page (maybe in the `Concepts` section of the documentation).

Reply:
I added a sentence in the introduction and created an empty page for tenancy in the concepts section. Creating the concepts and defining the tenancy behavior for the whole system I guess should be done in a separate PR.